### PR TITLE
chore: Update listing interface to match for metamask endpoint with optional properties

### PR DESCRIPTION
**Pull request by red-game-dev**
This is just to upgrade the interface

### CODE REVIEW

**Review by BrettCleary — APPROVED** [review]

File: packages/valist-sdk/src/typesApi.ts [hunk 1]
```diff
@@ -48,4 +48,9 @@ export interface Listing {
   project_meta: ProjectMetaApi;
   project_name: string;
   timestamp: number;
+  isVerified?: boolean;
+  isMetaMaskVerified?: boolean;
+  isMetMaskCompatible?: boolean;
```
Comment (awantoch) [inline packages/valist-sdk/src/typesApi.ts:53]:
```suggestion
  isMetaMaskMarketplace?: boolean;
```

File: packages/valist-sdk/src/typesApi.ts [hunk 2]
```diff
@@ -48,4 +48,9 @@ export interface Listing {
   project_meta: ProjectMetaApi;
   project_name: string;
   timestamp: number;
+  isVerified?: boolean;
+  isMetaMaskVerified?: boolean;
```
Comment (awantoch) [inline packages/valist-sdk/src/typesApi.ts:52]:
```suggestion
```

**Review by awantoch — CHANGES_REQUESTED** [review]

File: packages/valist-sdk/src/typesApi.ts (hunk 1 above)
Comment (red-game-dev) [reply packages/valist-sdk/src/typesApi.ts:53]: This change would need to be applied as well on hp-store, but atm it is fine to be pushed as it is optional 🙌

**Comment by BrettCleary** [comment]
closing for #402

**Pull request closed without merging**